src/FreqWindow.cpp

```diff
@@ -182,6 +183,36 @@ BEGIN_EVENT_TABLE(FrequencyPlotDialog, wxDialogWrapper)
    EVT_COMMAND(wxID_ANY, EVT_FREQWINDOW_RECALC, FrequencyPlotDialog::OnRecalc)
 END_EVENT_TABLE()
 
+// Permit approximately 46.60 minutes of selected samples at
```

Comment:
Is this calculation right?  I think it’s 23.3 minutes

Reply:
It is correct. Unusual is the left operand of the bit-shift being 2, it usually is 1 - which would then yield 23.3.
Changed to `1 << 27`, in case I make another push.